### ISSUE TITLE
LR for ssdlite-mobilenetv2 with pre-trained backbone

### ISSUE THREAD

**Issue by Etoye** [open-mmlab/mmdetection#6528]
Hi, I've found an ssd-lite mobilenet-v2 [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/ssd/ssdlite_mobilenetv2_scratch_600e_coco.py) for training the model from scratch, but would like to use it to re-train the model with my personal mobilenet_v2 backbone. I've converted it into mmdet format and, as far as I know, there are no errors in loading it. What is the optimal learning rate parameter and epochs will be? Same learning rate and 30 epochs, or reduce lr / use 1x or 2x schedule? (8GPUs)

**Comment by RangiLyu** [comment]
Hyperparameters are closely related to your data and the difficulty of the task. I can't give any good suggestions.

**Comment by Etoye** [comment]
@RangiLyu Thanks for the reply! Well, I am interested in COCO result reproduction of the scratch training, but with ImageNet and OID pre-trained backbones.

**Comment by Etoye** [comment]
I'll close it and experiment by myself with different schedules.
